src/update.ts

```diff
@@ -61,57 +65,22 @@ function prepareUpdate(update: Update): Promise<any> {
       counter += 1;
       return `${title}.md`;
     };
-
+    
     const options = {
-      owner: STATUS_REPO_OWNER,
-      path: "_posts",
+      owner: "siggame",
+      path: `_posts/${nextTitle()}`,
```

Comment:
So looking at nextTitle I'm not sure that this is going to do what's intended. From the code it looks like the appended number will always be 0.

Reply:
Adding the date and time and everything else looks fine, but the counter isn't going to work.